tests/php/_inc/lib/test_class.rest-api-endpoints.php

```diff
@@ -960,40 +960,78 @@ public function test_fetch_not_registered_widget_data() {
 	}
 
 	/**
-	 * Test changing the master user.
+	 * Test changing the master user without the owner parameter.
```

Comment:
Do these tests still belong here, or should they be moved along with the implementation of the endpoint they're testing?

Reply:
I don't think it's currently possible to implement these tests in the Connection package tests. These tests require creating users and saving them in the database. The Connection package tests do not run in a full WordPress environment with a database. Instead, they use the [WorDBless package](https://github.com/Automattic/wordbless), which provides WordPress core functions without a database.

So, I left the tests the require a WordPress environment in the Jetpack plugin's tests and added a few tests that do not require the  WordPress environment to the Connection package.